### PR TITLE
[STACK 0.1] Move to experimental

### PR DESCRIPTION
**Pull request by SimonBiggs**
Hi @sjswerdloff,

This is the first PR of attempting to merge in some of the work from `wlutz-arc`. This PR is aiming to start simple. It is moving all current "experimental apps" into the "experimental directory".

### CODE REVIEW

File: lib/pymedphys/_experimental/streamlit/apps/iviewdb.py [hunk 1]
```diff
@@ -19,7 +19,8 @@
 from pymedphys._imports import streamlit as st
 from pymedphys._imports import xmltodict
 
-from pymedphys._streamlit.utilities import dbf, misc
+from pymedphys._experimental.streamlit.utilities import dbf
+from pymedphys._streamlit.utilities import misc
```
Comment (sjswerdloff) [inline lib/pymedphys/_experimental/streamlit/apps/iviewdb.py:23]: Why is this not also in experimental?

**Review by sjswerdloff — APPROVED** [review]
But I would like to know why the one import of utilities is not through experimental

File: lib/pymedphys/_experimental/streamlit/apps/iviewdb.py (hunk 1 above)
Comment (SimonBiggs) [reply lib/pymedphys/_experimental/streamlit/apps/iviewdb.py:23]: This is a module that has its roots within the MetersetMap GUI originally:

https://github.com/pymedphys/pymedphys/blob/165dca77fa61d763f25f468dd6f8226a5f4603f9/lib/pymedphys/_streamlit/apps/metersetmap/_mosaiq.py#L20

https://github.com/pymedphys/pymedphys/blob/165dca77fa61d763f25f468dd6f8226a5f4603f9/lib/pymedphys/_streamlit/apps/metersetmap/_mosaiq.py#L37-L39

It is part of the non-experimental applications. It has some degree of coverage via e2e testing.

Comment (SimonBiggs) [reply lib/pymedphys/_experimental/streamlit/apps/iviewdb.py:23]: Here is it utilised within the Monaco section of MetersetMap:

https://github.com/pymedphys/pymedphys/blob/165dca77fa61d763f25f468dd6f8226a5f4603f9/lib/pymedphys/_streamlit/utilities/monaco.py#L9-L20

And the DICOM section:

https://github.com/pymedphys/pymedphys/blob/165dca77fa61d763f25f468dd6f8226a5f4603f9/lib/pymedphys/_streamlit/apps/metersetmap/_dicom.py#L90-L94

**Pull request merged**